Build failure: streamdeck-ui-4.1.2 (due to python3.12-pillow-10.4.0 failure)

### Steps To Reproduce

Steps to reproduce the behavior:
1. enable streamdeck ui 
```nix
  programs.streamdeck-ui = {
    enable = true;
    autoStart = true; # optional
  };
```

### Build log

```sh
# nixos-rebuild switch --upgrade
unpacking 2 channels...
warning: Git tree '/etc/nixos' is dirty
building the system configuration...
warning: Git tree '/etc/nixos' is dirty
trace: evaluation warning: The ‘gnome.gnome-software’ was moved to top-level. Please use ‘pkgs.gnome-software’ directly.
trace: evaluation warning: The ‘gnome.mutter’ was moved to top-level. Please use ‘pkgs.mutter’ directly.
error: builder for '/nix/store/ai8pqvzjg4b4avmhnmwwjl30vhfbyz0v-streamdeck-ui-4.1.2.drv' failed with exit code 1;
       last 25 log lines:
       >
       > tests/gui/test_button_actions.py:46
       >   /build/source/tests/gui/test_button_actions.py:46: PytestUnknownMarkWarning: Unknown pytest.mark.serial - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.serial
       >
       > tests/gui/test_config_backup_dialog.py:9
       >   /build/source/tests/gui/test_config_backup_dialog.py:9: PytestUnknownMarkWarning: Unknown pytest.mark.serial - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.serial
       >
       > tests/gui/test_config_backup_dialog.py:32
       >   /build/source/tests/gui/test_config_backup_dialog.py:32: PytestUnknownMarkWarning: Unknown pytest.mark.serial - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.serial
       >
       > tests/gui/test_config_backup_dialog.py:55
       >   /build/source/tests/gui/test_config_backup_dialog.py:55: PytestUnknownMarkWarning: Unknown pytest.mark.serial - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.serial
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > FAILED tests/test_filter.py::test_image_filter[smile.png] - TypeError: argument 1 must be 2-item sequence, not generator
       > FAILED tests/test_filter.py::test_image_filter[smile.jpg] - TypeError: argument 1 must be 2-item sequence, not generator
       > FAILED tests/test_filter.py::test_image_filter[smile.svg] - TypeError: argument 1 must be 2-item sequence, not generator
       > FAILED tests/test_filter.py::test_image_filter[dog.gif] - TypeError: argument 1 must be 2-item sequence, not generator
       > FAILED tests/test_filter.py::test_pipeline - TypeError: argument 1 must be 2-item sequence, not generator
       > =================== 5 failed, 92 passed, 6 warnings in 6.57s ===================
       For full logs, run 'nix log /nix/store/ai8pqvzjg4b4avmhnmwwjl30vhfbyz0v-streamdeck-ui-4.1.2.drv'.
error: 1 dependencies of derivation '/nix/store/6ffv813j4pp9f1275rs8iakjm0figzyy-X-Restart-Triggers-systemd-udevd.drv' failed to build
error: 1 dependencies of derivation '/nix/store/c50il5lxg4l6bh8mlffffly6s4pqpsga-hwdb.bin.drv' failed to build
error: 1 dependencies of derivation '/nix/store/y86afvnj78vsjg7lc1flgbzsyjmzzza9-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/nwys9acfn7qj84a9nkyqnajxcwrs6007-nixos-system-Ifrit-24.11.20240906.574d1ea.drv' failed to build

# nix log /nix/store/ai8pqvzjg4b4avmhnmwwjl30vhfbyz0v-streamdeck-ui-4.1.2.drv
warning: The interpretation of store paths arguments ending in `.drv` recently changed. If this command is now failing try again with '/nix/store/ai8pqvzjg4b4avmhnmwwjl30vhfbyz0v-streamdeck-ui-4.1.2.drv^*'
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/5p4c0lzm8qgfaf9d2wd9gfshq0hwl25p-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/udev/60-streamdeck.rules
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
* Getting build dependencies for wheel...
* Building wheel...
Successfully built streamdeck_linux_gui-4.1.1-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/streamdeck_linux_gui-4.1.1...OK
Repacking wheel as ./streamdeck_linux_gui-4.1.1-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for streamdeck_linux_gui-4.1.1-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Executing pypaInstallPhase
Successfully installed streamdeck_linux_gui-4.1.1-py3-none-any.whl
substituteStream() in derivation streamdeck-ui-4.1.2: WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'scripts/streamdeck.service')
Copying '/nix/store/sp1yxbli378qvvb3d4vs2lh6hs89apcg-streamdeck-ui.desktop/share/applications/streamdeck-ui.desktop' into '/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/share/applications'
Copying '/nix/store/89si9yqjlb9akaim24h6pdjy6dmisgqk-streamdeck-ui-noui.desktop/share/applications/streamdeck-ui-noui.desktop' into '/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/share/applications'
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: qtOwnPathsHook
@nix { "action": "setPhase", "phase": "qtOwnPathsHook" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: gappsWrapperArgsHook
@nix { "action": "setPhase", "phase": "gappsWrapperArgsHook" }
Running phase: dropIconThemeCache
@nix { "action": "setPhase", "phase": "dropIconThemeCache" }
Running phase: dropIconThemeCache
@nix { "action": "setPhase", "phase": "dropIconThemeCache" }
Running phase: fixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
shrinking RPATHs of ELF executables and libraries in /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2
checking for references to /build/ in /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2...
moving /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/lib/systemd/user/* to /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/share/systemd/user
patching script interpreter paths in /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2
stripping (with command strip and flags -S -p) in  /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/lib /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/bin
shrinking RPATHs of ELF executables and libraries in /nix/store/rvwxhj4y483gzis5fd04x9i8wjsbsxfs-streamdeck-ui-4.1.2-dist
checking for references to /build/ in /nix/store/rvwxhj4y483gzis5fd04x9i8wjsbsxfs-streamdeck-ui-4.1.2-dist...
patching script interpreter paths in /nix/store/rvwxhj4y483gzis5fd04x9i8wjsbsxfs-streamdeck-ui-4.1.2-dist
Rewriting #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/bin/python3.12 to #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5
wrapping `/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/bin/streamdeck'...
Rewriting #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/bin/python3.12 to #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5
wrapping `/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/bin/streamdeckc'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
rewriting symlink /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/lib/systemd/user to be relative to /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
============================= test session starts ==============================
platform linux -- Python 3.12.5, pytest-8.3.2, pluggy-1.5.0
PySide6 6.7.2 -- Qt runtime 6.7.2 -- Qt compiled 6.7.2
rootdir: /build/source
configfile: pyproject.toml
plugins: qt-4.4.0, mock-3.14.0
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for streamdeck_linux_gui-4.1.1-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Executing pypaInstallPhase
Successfully installed streamdeck_linux_gui-4.1.1-py3-none-any.whl
substituteStream() in derivation streamdeck-ui-4.1.2: WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'scripts/streamdeck.service')
Copying '/nix/store/sp1yxbli378qvvb3d4vs2lh6hs89apcg-streamdeck-ui.desktop/share/applications/streamdeck-ui.desktop' into '/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/share/applications'
Copying '/nix/store/89si9yqjlb9akaim24h6pdjy6dmisgqk-streamdeck-ui-noui.desktop/share/applications/streamdeck-ui-noui.desktop' into '/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/share/applications'
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: qtOwnPathsHook
@nix { "action": "setPhase", "phase": "qtOwnPathsHook" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: gappsWrapperArgsHook
@nix { "action": "setPhase", "phase": "gappsWrapperArgsHook" }
Running phase: dropIconThemeCache
@nix { "action": "setPhase", "phase": "dropIconThemeCache" }
Running phase: dropIconThemeCache
@nix { "action": "setPhase", "phase": "dropIconThemeCache" }
Running phase: fixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
shrinking RPATHs of ELF executables and libraries in /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2
checking for references to /build/ in /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2...
moving /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/lib/systemd/user/* to /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/share/systemd/user
patching script interpreter paths in /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2
stripping (with command strip and flags -S -p) in  /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/lib /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/bin
shrinking RPATHs of ELF executables and libraries in /nix/store/rvwxhj4y483gzis5fd04x9i8wjsbsxfs-streamdeck-ui-4.1.2-dist
checking for references to /build/ in /nix/store/rvwxhj4y483gzis5fd04x9i8wjsbsxfs-streamdeck-ui-4.1.2-dist...
patching script interpreter paths in /nix/store/rvwxhj4y483gzis5fd04x9i8wjsbsxfs-streamdeck-ui-4.1.2-dist
Rewriting #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/bin/python3.12 to #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5
wrapping `/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/bin/streamdeck'...
Rewriting #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5/bin/python3.12 to #!/nix/store/pgb120fb7srbh418v4i2a70aq1w9dawd-python3-3.12.5
wrapping `/nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/bin/streamdeckc'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
rewriting symlink /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2/lib/systemd/user to be relative to /nix/store/25fhfvs8r4i71f24wgdm7cjf2n9vz13k-streamdeck-ui-4.1.2
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
============================= test session starts ==============================
platform linux -- Python 3.12.5, pytest-8.3.2, pluggy-1.5.0
PySide6 6.7.2 -- Qt runtime 6.7.2 -- Qt compiled 6.7.2
rootdir: /build/source
configfile: pyproject.toml
plugins: qt-4.4.0, mock-3.14.0
collecting ... The 'fc-list' command is not available on your system. Using fallback fonts.
^Mcollected 97 items                                                             

tests/api/test_button_state.py ...................                       [ 19%]
tests/api/test_default_state.py ..                                       [ 21%]
tests/api/test_device_state.py ....                                      [ 25%]
tests/api/test_pages.py .......                                          [ 32%]
tests/cli/test_commands.py ...........                                   [ 44%]
tests/gui/test_button_actions.py ...                                     [ 47%]
tests/gui/test_config_backup_dialog.py ...                               [ 50%]
tests/gui/test_menu.py .....                                             [ 55%]
tests/gui/test_page_actions.py ...                                       [ 58%]
tests/gui/test_switch_page.py ..                                         [ 60%]
tests/modules/test_fonts.py ...                                          [ 63%]
tests/modules/test_keyboard.py ..................                        [ 82%]
tests/test_config.py ......                                              [ 88%]
tests/test_dimmer.py ....                                                [ 92%]
tests/test_filter.py .FFFFF                                              [ 98%]
tests/test_stream_deck_monitor.py .                                      [100%]

=================================== FAILURES ===================================
_________________________ test_image_filter[smile.png] _________________________

image = 'smile.png'

    @pytest.mark.parametrize("image", ["smile.png", "smile.jpg", "smile.svg", "dog.gif"])
    def test_image_filter(image: str):
        size = (72, 72)
        pipe = pipeline.Pipeline()
    
        filter = empty_filter.EmptyFilter()
        filter.initialize(size)
    
        pipe.add(filter)
        time = Fraction(0)
    
        filter = image_filter.ImageFilter(get_asset(image))
>       filter.initialize(size)

tests/test_filter.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
streamdeck_ui/display/image_filter.py:90: in initialize
    frame = frame.resize((int(v * scale_factor) for v in frame.size), Image.LANCZOS)
/nix/store/cbvla7s1jq36kgf4g3gsbap06f9hs8lz-python3.12-pillow-10.4.0/lib/python3.12/site-packages/PIL/Image.py:2305: in resize
    im = im.resize(size, resample, box)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

```

### Additional context

x64 system
nixos-unstable

### Notify maintainers

@prikhi
@Majiir
@mweinelt 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
# nix-shell -p nix-info --run "nix-info -m"
fetching path input 'path:/nix/store/5w3dp0m37794iffsbm9vd9f1xmmhda6i-source'
 - system: `"x86_64-linux"`
 - host os: `Linux 6.10.6, NixOS, 24.11 (Vicuna), 24.11.20240906.574d1ea`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.0
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /root/.config/nix/nix.conf:/root/.config/kdedefaults/nix/nix.conf:/etc/xdg/nix/nix.conf:/root/.local/share/flatpak/exports/etc/xdg/nix/nix.conf:/var/lib/flatpak/exports/etc/xdg/nix/nix.conf:/root/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/root/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/root/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/cffwhlcs83chmzkikcfd3mchq3lklnfg-lix-2.91.0/share`
 - channels(root): `"nixos-23.11, nixos-unstable"`
 - nixpkgs: `/nix/store/5w3dp0m37794iffsbm9vd9f1xmmhda6i-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/st/streamdeck-ui/package.nix
+++ b/pkgs/by-name/st/streamdeck-ui/package.nix
@@ -11,35 +11,28 @@
   xvfb-run,
 }:
 
-let
-  # There are breaking changes between 6 and 7
-  importlib-metadata_6 = python3Packages.importlib-metadata.overrideAttrs (_: rec {
-    version = "6.9.0";
-    src = fetchPypi {
-      pname = "importlib_metadata";
-      inherit version;
-      hash = "sha256-6Ky1I8M1qRgiZ04Um0bAOZ7E0yjE0fbknCc9pf8CAbk=";
-    };
-  });
-in
-
 python3Packages.buildPythonApplication rec {
   pname = "streamdeck-ui";
-  version = "4.1.2";
+  version = "4.1.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     repo = "streamdeck-linux-gui";
     owner = "streamdeck-linux-gui";
     rev = "v${version}";
-    hash = "sha256-CSsFPGnKVQUCND6YOA9kfO41KS85C57YL9LcrWlQRKo=";
+    hash = "sha256-KpsW3EycYRYU5YOg7NNGv5eeZbS9MAikj0Ke2ybPzAU=";
   };
 
-  format = "pyproject";
+  pythonRelaxDeps = [
+    "importlib-metadata"
+    "pillow"
+  ];
 
-  pythonRelaxDeps = [ "pillow" ];
+  build-system = [
+    python3Packages.poetry-core
+  ];
 
   nativeBuildInputs = [
-    python3Packages.poetry-core
     copyDesktopItems
     qt6.wrapQtAppsHook
     wrapGAppsHook3
@@ -56,7 +49,7 @@ python3Packages.buildPythonApplication rec {
       pyside6
       streamdeck
       xlib
-      importlib-metadata_6
+      importlib-metadata
       evdev
     ]
     ++ lib.optionals stdenv.isLinux [ qt6.qtwayland ];
@@ -128,6 +121,7 @@ python3Packages.buildPythonApplication rec {
   ];
 
   meta = {
+    changelog = "https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v${version}";
     description = "Linux compatible UI for the Elgato Stream Deck";
     downloadPage = "https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/";
     homepage = "https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/";

